machines/4x4/Makefile.machine.include

```diff
@@ -8,3 +8,4 @@ BSG_MACHINE_DRAM_BANK_SIZE_WORDS      = 134217728
 BSG_MACHINE_DRAM_INCLUDED             = 1
 BSG_MACHINE_MAX_EPA_WIDTH             = 28
 BSG_MACHINE_MEM_CFG                   = e_mem_cfg_default
+BSG_MACHINE_BJ_TRACE_EN               = 0
```

Comment:
can we call this something other than BJ...? I think Branch Trace is just fine.

Reply:
The reason is that you could be offending some people.. gotta be careful when picking acronyms..

Reply:
Oh okay! I'll change it to branch tarce.